main.py

```diff
@@ -1,7 +1,134 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from typing import Any
```

Comment:
must be a blank line between since typing is build in library and django is external package